index-scheduler/src/index_mapper.rs

```diff
@@ -228,3 +231,22 @@ impl IndexMapper {
         &self.indexer_config
     }
 }
+
+/// A heed codec for value of struct Uuid.
+pub struct UuidCodec;
+
+impl<'a> BytesDecode<'a> for UuidCodec {
+    type DItem = Uuid;
+
+    fn bytes_decode(bytes: &'a [u8]) -> Option<Self::DItem> {
+        bytes.try_into().ok().map(Uuid::from_bytes)
+    }
+}
+
+impl BytesEncode<'_> for UuidCodec {
+    type EItem = Uuid;
+
+    fn bytes_encode(item: &Self::EItem) -> Option<Cow<[u8]>> {
+        Some(Cow::Borrowed(item.as_bytes()))
+    }
+}
```

Comment:
Would it be possible to move this impl to another distinct file, named `uuid_codec`, maybe?

Reply:
I have moved this to a new file named uuid_codec.rs.
I actually forgot to ask where should this file be situated? Currently it is in the same folder as `index_mapper.rs`. Is this fine or should it moved to different folder?

Reply:
That's perfect, thank you!